packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -1233,6 +1241,12 @@ export function warnForUnmatchedText(textNode: Text, text: string) {
   }
 }
 
+export function getIsRenderingInDEV() {
```

Comment:
What value does this function add? Looks like it's only called from 4 places, that themselves are already inside of a `__DEV__` conditional. Seems like we could just remove this indirection? (Maybe it doesn't matter since it would get inlined anyway but..)